pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -231,15 +231,20 @@ class Build {
 
         def buildNumber = versionData.build
 
+        String releaseType = "Nightly"
+        if (buildConfig.RELEASE) {
+            releaseType = "Release"
+        }
+
         def installerJob = context.build job: "build-scripts/release/create_installer_linux",
                 propagate: true,
                 parameters: [
                         context.string(name: 'UPSTREAM_JOB_NUMBER', value: "${env.BUILD_NUMBER}"),
                         context.string(name: 'UPSTREAM_JOB_NAME', value: "${env.JOB_NAME}"),
                         context.string(name: 'FILTER', value: "${filter}"),
+                        context.string(name: 'RELEASE_TYPE', value: "${releaseType}"),
                         context.string(name: 'VERSION', value: "${versionData.version}"),
                         context.string(name: 'MAJOR_VERSION', value: "${versionData.major}"),
-                        context.string(name: 'JVM', value: "${buildConfig.VARIANT}"),
```

Comment:
Why is this removed?

Reply:
The type of JVM (Hotspot, OpenJ9 etc.) is now inferred from the filename of the artifacts copied into the workspace.